test/list.spec.js

```diff
@@ -304,3 +304,32 @@ describe('Linked List', function () {
             });
     });
 });
+
+describe('Pre initialized Linked List', function () {
+
+    // Utility function to populate the list with dummy data.
+    // The number of nodes added will be specified by the 'numNodes'
+    // parameter.
+    var populateArray = function (arr, numItems) {
+        for (var i = 0; i < numItems; i++) {
+            arr.push('test item ' + (i + 1));
+        }
+    };
+
+
+    beforeEach(function () {      
+        var arr = [];
+        populateArray(arr, 5);  
+        list = new LinkedList(arr);
+    });
+
+
+    afterEach(function () {
+        list = null;
+    });
+
+    it('initially contains five items', function () {
+        expect(list.isEmpty()).to.be.false;
+        expect(list.getSize()).to.equal(5);
+    });
+});
```

Comment:
Since this only contains one test, we don't explicitly need the `beforeEach` and `afterEach` blocks.  Also, since we are specifically testing the linked list construction, it would make sense to move that inside the test itself.  The way it is now, will certainly work, and it does set us up ok if we want/need to add additional tests down the the road.